cg/utils/utils.py

```diff
@@ -1,9 +1,22 @@
 """Helper functions."""
 
-from typing import Optional, List
+from typing import List, Optional
 
 
 def get_string_from_list_by_pattern(strings: List[str], pattern: str) -> Optional[str]:
     """Returns the full string from a list given a specific pattern."""
 
     return next((string for string in strings if pattern in string), None) if strings else None
+
+
+def build_command_from_dict(options: dict, exclude_true: bool = False) -> List[str]:
+    """Returns a command list of strings given a dictionary with arguments."""
+    formatted_options: list = []
+    for key, val in options.items():
```

Comment:
Maybe something more descriptive like this?
```suggestion
    for flag, option in options.items():
```

Reply:
changed to option_name, option_value in previous comment.